Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Use official CentOS Python 2.7 image
```

Comment:
@rollandf , shoulds we use python 3.6 for it?

Reply:
Python 2.7 is not supported, so yes.

Reply:
Hi, i'm not sure where you see this line.
In the files I commited the dicker uses python 3.7

Reply:
Maybe after rebasing it will be removed.
You can see another reference at line 3.